src/plugin.js

```diff
@@ -320,6 +328,10 @@ install.mutations[VUEXFIRE_ARRAY_MOVE] = function (state, payload) {
   array.splice(payload.newIndex, 0, array.splice(payload.index, 1)[0])
 }
 
+install.mutations[VUEXFIRE_INIT_WITH_VALUE] = function (state, payload) {
+  state[payload.key] = payload.value
```

Comment:
Here is where the method `getModuleState` should be called `getModuleState(state, module)[key] = value`